sdk/monitor/Microsoft.OpenTelemetry.Exporter.AzureMonitor/src/TelemetryPartA.cs

```diff
@@ -0,0 +1,92 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Globalization;
+using OpenTelemetry.Resources;
+using OpenTelemetry.Trace;
+using Microsoft.OpenTelemetry.Exporter.AzureMonitor.Models;
+
+namespace Microsoft.OpenTelemetry.Exporter.AzureMonitor
+{
+    internal class TelemetryPartA
```

Comment:
I think it's really important for these classes to have a summary. Only persons with intimate knowledge of the Common Schema are going to understand what PartA, B, C mean.